docs/sphinx/install.rst

```diff
@@ -1,128 +1,183 @@
+guidance for application development on a remote host where CUDA Quantum is installed.
 
-Install the Docker Image
-++++++++++++++++++++++++++++++++++++
+.. _install-cuda-quantum:
 
-Docker images for all CUDA Quantum releases are available on the `NGC Container Registry`_.
-To download images from NGC, please follow the following steps if you have not done so already:
+Local Installation
+------------------------------------
 
-- `Create an account <https://ngc.nvidia.com/signin>`__
-- `Sign in <https://ngc.nvidia.com/signin>`__ to access your account and go to `Setup <https://ngc.nvidia.com/setup>`__.
-- Click on `Get API Key` and generate a new key (this will invalidate any existing keys).
-- Follow the instructions that appear to use that key to log in to the NGC registry using Docker.
+A fully featured CUDA Quantum installation including all C++ and Python tools is available as a 
+`Docker image`_. A `Singularity`_ container can easily be created based on these images. Additionally, we distribute pre-built `Python wheels`_ via PyPI.
+If you would like to build CUDA Quantum from source instead, please follow the instructions on the `CUDA Quantum GitHub repository`_.
 
-Once you have done so, run `docker login nvcr.io` (again) to confirm you can authenticate with the registry. 
-You should see a message "Login Succeeded".
+.. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
 
-.. _NGC Container Registry: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuda-quantum
+If you are unsure which option suits you best, we recommend using our `Docker image`_ to develop your applications in a controlled environment that does not depend on, or interfere with, other software
+that is installed on your system.
+
+.. _install-docker-image:
 
-In addition to publishing stable releases, we also publish docker images whenever we update 
-certain branches of our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`_.
-These images are published in a separate location `nvidia/nightly` on NGC, as well as on GitHub.
+Docker image
+++++++++++++++++++++++++++++++++++++
+
+To download and use our Docker images, you will need to install and launch the Docker engine. 
+If you do not already have Docker installed on your system, you can get it by downloading and installing `Docker Desktop <https://docs.docker.com/get-docker/>`_. 
+If you do not have the necessary administrator permissions to install software on your machine, 
+take a look at the section below on how to use `Singularity`_ instead.
+
+Docker images for all CUDA Quantum releases are available on the `NGC Container Registry`_.
+In addition to publishing `stable releases <https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuda-quantum/tags>`__, 
+we also publish docker images whenever we update certain branches on our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`_.
```

Comment:
nit: Docker is capitalized nearly everywhere else, so it probably should be capitalized here, too.

Reply:
Yes, that's an oversight - found another one of that, too. Oddly enough, the spellcheck recognizes the lowercase version as well ...